src/tests/example.spec.js

```diff
@@ -28,3 +24,71 @@ test.describe('Saucedemo app basic tests', () => {
         await expect(app.shoppingCart.cartItems).not.toBeAttached();
     });
 });
+
+test.describe('Saucedemo Unit 10 tests', () => {
+    const sortOption = [
+        {sort: "za", sortByName: (inventoryInitialOrder) => {
+            return inventoryInitialOrder.sort((a, b) => b.localeCompare(a));
+            }},
+        {sort: "az", sortByName: (inventoryInitialOrder) => {
+                return inventoryInitialOrder.sort();
+            }},            
+        {sort: "lohi", sortByPrice: (inventoryInitialOrder) => {
+                return inventoryInitialOrder.sort((a, b) => Number(a) - Number(b));
+            }},
+        {sort: "hilo", getExpectedResultPrice: (productPriceInitialOrderToNumbers) => {
```

Comment:
This should look like this, right?
```suggestion
        {sort: "hilo", sortByPrice: (inventoryInitialOrder) => {
```